easybuild/easyconfigs/p/Pychopper/Pychopper-0.4.0-intel-2018a-Python-3.7.0.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonPackage'
+
+name = 'Pychopper'
+version = '0.4.0'
+versionsuffix = '-Python-3.7.0'
+
+homepage = 'https://github.com/nanoporetech/pychopper'
+description = """A tool to identify full length cDNA reads.
+Primers have to be specified as they are on the forward strand."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/nanoporetech/pychopper/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['parasail.patch']
+checksums = [
+    '5cdc6bdca0723d70072125fb9035692cc6c069b58b5797955a2c0425da8ede3f',  # v0.4.0.tar.gz
+    '569505eac4a2c9b8433376089819bab773213067113eec38640597fb87e481ba',  # parasail.patch
+]
+
+dependencies = [
+    ('Python', '3.7.0'),
```

Comment:
@HPC-UniOldenburg - this version of `Python` is not available for the `intel/2018a` toolchain. It would be best either to use a different toolchain, or to use one of the `Python` versions that is available.